scripts/chatty_news.js

```diff
@@ -0,0 +1,512 @@
+/*!
+* Adds recent news headlines to top right of chatty by reading the Shack RSS feed.
+* Can add additional description line from RSS feed to Shacknews article posts.
+* Can add a 'wallpaper' background to the chatty page which reflects the main image of the most recent article.
+*/
+
+settingsLoadedEvent.addHandler(function()
+{
+    if (getSetting("enabled_scripts").contains("chatty_news"))
+    {
+        chattynews = {
+            LOCALSTORAGE_LASTIMAGE: "chatty_news_last_image",
+            LOCALSTORAGE_LASTNEWS: "chatty_news_last_news_url",
+            scriptStartTime: 0,
+            tintback: true,
+            
+            install: function(){
+                if(document.getElementById("SNCNheadlines")!=null)
+                    return; //if this div exists, we've already been installed
+                if((document.location).toString().indexOf('/chatty')>-1){
+                    
+                    chattynews.scriptStartTime = (new Date()).getTime();
+                    
+                    var backcolor = "black";
+                    var forecolor = "#ddd";//"white";
+                    var styletext = `#SNCNbackimg {
```

Comment:
Don't embed styling like this if you can avoid it. It makes debugging and iterative testing more difficult. We put CSS in .`/styles/*.css`.